.travis.yml

```diff
@@ -20,3 +20,18 @@ before_script:
   # http://docs.travis-ci.com/user/database-setup/#MySQL
   - sudo /etc/init.d/postgresql stop
    # - sudo /etc/init.d/mysql stop
+
+cache: pip
+
+deploy:
```

Comment:
You should move this to a separate stage.

Reply:
why? deploy starting only for tag.

Reply:
Because it will hit all matrix jobs. I know that it's just one now, but you'll add more. Also, don't interfere with tests env. It must be separated into its own job.